develop/docs/dlq.md

```diff
@@ -0,0 +1,35 @@
+# Managing DLQ Using tdbg
+When a task in the Temporal server encounters a terminal error, it may be enqueued in the Dead Letter Queue (DLQ). 
+To effectively manage these DLQ messages, follow these steps:
+
+1. There is a metric `dlq_writes`, which will be incremented each time a message is enqueued to the DLQ.
+2. Search `Task enqueued to DLQ` in logs and see task details. Get the DLQ message ID from this log line.
+3. From the log trace, you can identify the offending namespace, task type, source cluster, and target cluster of the task.
```

Comment:
I would consolidate this with step 2. Also, I'd only include the task type, source cluster, target cluster and message ID. Also, I'd define that source cluster and target cluster are only different for replication tasks.